tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -1401,4 +1418,68 @@ describe(__filename, () => {
       ).not.toHaveClass('UserProfileEditPicture-file--has-focus');
     });
   });
+
+  describe('Tests for accounts branding', () => {
+    let isMzaBrandingMock;
+    let dateMock;
+
+    beforeEach(() => {
+      isMzaBrandingMock = jest
+        .spyOn(constants, 'isMzaBranding')
+        .mockReturnValue(true);
+    });
+
+    afterEach(() => {
+      isMzaBrandingMock.mockRestore();
+      dateMock?.mockRestore();
+    });
+    it('renders the Mozilla Accounts Branding', () => {
+      renderForCurrentUser();
+
+      expect(
+        screen.getByRole('link', {
+          name: 'Manage Mozilla Accounts…',
+        }),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByTextAcrossTags(
+          'You can change your email address on Mozilla Accounts. Need help?',
+        ),
+      ).toBeInTheDocument();
+    });
+
+    it('renders the Firefox Accounts Branding', () => {
+      isMzaBrandingMock = jest
+        .spyOn(constants, 'isMzaBranding')
+        .mockReturnValue(false);
+      renderForCurrentUser();
+
+      expect(
+        screen.getByRole('link', {
+          name: 'Manage Firefox Accounts…',
+        }),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByTextAcrossTags(
+          'You can change your email address on Firefox Accounts. Need help?',
+        ),
+      ).toBeInTheDocument();
+    });
+
+    it('returns false if the date is earlier than MZA_LAUNCH_DATETIME', () => {
+      isMzaBrandingMock.mockRestore();
+      const before = new Date();
+      before.setTime(MZA_LAUNCH_DATETIME.getTime() - 1000);
+      dateMock = jest.spyOn(global, 'Date').mockImplementation(() => before);
+      expect(isMzaBranding()).toBe(false);
+    });
+
+    it('returns true if the date is earlier than MZA_LAUNCH_DATETIME', () => {
+      isMzaBrandingMock.mockRestore();
+      const after = new Date();
+      after.setTime(MZA_LAUNCH_DATETIME.getTime() + 1000);
+      dateMock = jest.spyOn(global, 'Date').mockImplementation(() => after);
+      expect(isMzaBranding()).toBe(true);
+    });
```

Comment:
move these into their own `describe` block, for testing `isMzaBranding` specifically.